src/css/Mdx.css

```
@@ -0,0 +1,206 @@
+.blog-post-body > * + * {
+	margin-top: 1.1rem;
+}
+
+.blog-post-body p {
+	font-weight: 300;
+}
+
+.blog-post-body a {
+	color: var(--color-accent);
+	text-decoration: none;
+	border-bottom: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)